backend/composer/services/cs_ingestion/helpers/overwritable_helper.py

```diff
@@ -10,25 +10,35 @@
 logger_service = LoggerService()
 
 
-def get_overwritable_statements(statements_list: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
-    overwritable_statements = [
+def get_overwritable_and_new_statements(statements_list: List[Dict[str, Any]], disable_overwrite: bool=False) -> List[Dict[str, Any]]:
+    overwritable_and_new_statements = [
         statement for statement in statements_list
-        if not has_invalid_sentence(statement) and not has_invalid_statement(statement)
+        if not has_invalid_sentence(statement, disable_overwrite) and not has_invalid_statement(statement, disable_overwrite)
```

Comment:
Because it depends on both the utils here. As you can see after running the script (from the second video). It updates the Connectivity Statement to len 426 (from 423), and Sentences to be 466 (from 463). Essentially adding 3. 

here Sentences are back (3 deleted, 3 ingested). However the Statements are not - I deleted 43, ingested Connectivity statements are only 3, (40 more are not ingestable). 

@afonsobspinto what do you think? should we update the logic here? (Or is it the case that the test case I used was a peculiar/non-realistic one).

Reply:
The 40 statements were not ingested because the sentences were not in COMPOSE_NOW? If that's the case it sounds correct to me  (@ddelpiano can you double check here, what are the conditions to a statement be ingested when the disable override is set to true)